### ISSUE TITLE
Packaging problem and 404 error 

### ISSUE THREAD

**Issue by Kajilenu** [ruyo/VRM4U#43]
I'm trying to access https://github.com/ruyo/UnrealEngine_VRM4UPlugin but I keep getting 404 error even though my epic and github account is linked.

**Comment by ruyo** [comment]
Hi.
Is there an icon in your account information?
Perhaps your account is not linked.

![image](https://user-images.githubusercontent.com/6472835/95365575-4b733b80-090d-11eb-9bc4-9df01268d684.png)

**Comment by Kajilenu** [comment]
It says connected on the Epic side but the organizations isn't showing up....am I missing a step?
[image](https://user-images.githubusercontent.com/72482164/95410292-20f9a080-0956-11eb-88a9-c2c7bea0d439.png)

**Comment by Xi-v** [comment]
same problem

**Comment by ruyo** [comment]
Check to see if your account is linked from the github page.
You will probably need to authenticate with your email address.

The trouble I've seen before is that sometimes it doesn't work when you either enter your github account name or your email address when linking.

**Comment by Kajilenu** [comment]
Okay I finally got it working. So the problem was I didn't get notification of an invite to join the organization when I connected on the Epic side, thus I was not aware of having to accept an invitation. I had to open up Epic's repository page then there was an pop up that appeared telling me an invite was sent and the option to accept the invite.

Long story short is that there's an extra step of accepting the organization invite on Github after connecting on Epic XD Thanks!
